### PR TITLE
Lodash to native

### PR DESCRIPTION
**Pull request by Page-**

### CODE REVIEW

File: lf-to-abstract-sql.ometajs [hunk 1]
```diff
@@ -214,7 +214,7 @@ export ometa LF2AbstractSQL <: SBVRCompilerLibs {
 		AddRelationship(baseTable, factType.slice(1), fkName, references)
 		GetResourceName(factType):factTypeResourceName
 		{
-			_.each($elf.synonymousForms[factTypeResourceName], function(synForm) {
+			_.forEach($elf.synonymousForms[factTypeResourceName], function(synForm) {
```
Comment (dfunckt) [inline lf-to-abstract-sql.ometajs:217]: Did you mean

```suggestion
			$elf.synonymousForms[factTypeResourceName].forEach(function(synForm) {
```

?

File: lf-to-abstract-sql.ometajs [hunk 2]
```diff
@@ -872,8 +872,8 @@ export ometa LF2AbstractSQL <: SBVRCompilerLibs {
 		]
 		{{}}:hasDependants
 		{
-			_.each(this.tables, function(table) {
-				_.each(table.fields, function(field) {
+			_.forEach(this.tables, function(table) {
```
Comment (dfunckt) [inline lf-to-abstract-sql.ometajs:875]: Did you mean

```suggestion
			this.tables.forEach(function(table) {
```

here and below?

File: lf-to-abstract-sql.ometajs (hunk 1 above)
Comment (Page-) [reply lf-to-abstract-sql.ometajs:217]: Nope, this is an object - I included it as standardizing on forEach to match the native version

File: lf-to-abstract-sql.ometajs (hunk 2 above)
Comment (Page-) [reply lf-to-abstract-sql.ometajs:875]: Nope, this is an object - I included it as standardizing on forEach to match the native version

**Pull request merged**